<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>661</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>Receive Text</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="networkButton01">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Config IP / Port</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox01">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>621</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QTextEdit" name="textEdit01">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>591</width>
         <height>151</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="closeButton01">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>170</y>
         <width>85</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clearButton">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>170</y>
         <width>85</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="networkButton02">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Config IP / Port</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox02">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>621</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="displayPic">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>351</width>
         <height>171</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>Picture Display *HERE*</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="closeButton02">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>200</y>
         <width>85</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>Receive Picture</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TCP Receive Node</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
